Core/Graphics/Source/UWP/GraphicsPlatform.cpp

```diff
@@ -0,0 +1,32 @@
+#include <Babylon/Graphics.h>
+#include <Babylon/GraphicsPlatform.h>
+#include "../GraphicsImpl.h"
+
+#include <winrt/Windows.Graphics.Display.h>
+
+using namespace winrt;
+using namespace Windows::Graphics::Display;
+
+namespace Babylon
+{
+    void GraphicsImpl::ConfigureBgfxPlatformData(const GraphicsConfiguration& config, bgfx::PlatformData& pd)
```

Comment:
I think I was imagining we'd have two versions of this, just like you do for the headers (`ICoreWindow*` vs. `ISwapChainPanel*`), and then they could just set the `pd.nwh` to the correct type.

Reply:
That's fine, I originally opted to share it as the implementations seemed identical, but this makes sense.

Reply:
Added a separate implementations for the different window types.